### DISCUSSION TITLE
is it worth working on getting defi arbitrages and liquidation opportunities in this competitive area which will be more competitive in the future?

### DISCUSSION THREAD

**Discussion by elia07** [Cyfrin/foundry-full-course-cu#2357] [General]
is it worth working on getting defi arbitrages and liquidation opportunities in this competitive area which will be more competitive in the future?

sir Patrick 
thanks for your always top-notch tutorials, I have learned most of my knowledge from you in this space.
recently I have been thinking about developing a code base or an infrastructure to try to do some arbitrage and try to find some liquidation opportunity in defi protocols.
based on my plans it takes 4 months to develop an MVP for it (I want to use Moralis SDK and if not the time extends to at least 8 months), if possible I want to know the opinion of an expert like you about this topic, does it work in these competitive days ?!

bb or !bb  :)

**Comment by EngrPips** [comment]
Hello @elia07, I trust @PatrickAlphaC  will reply to this as soon as he is available.

**Reply by elia07** [reply]
I am sure he will

**Reply by EngrPips** [reply]
Indeed
